### ISSUE TITLE
Program to find square root without the sqrt function.

### ISSUE THREAD

**Issue by H9660** [H9660/Hacktoberfest_2022#2]
You may use any langauge of your choice (C++, C, Python, Java).

**Comment by GulamMd** [comment]
Hi i am taking it...

**Comment by H9660** [comment]
Dont push the changes to the master branch directly.

**Comment by mehraankush** [comment]
yeah i will make another branch then will make the changes and push it. assign
 it to me

**Comment by gaurang7goel** [comment]
Hi, is this issue assigned to someone or open for work?

**Comment by roshan2152** [comment]
Hey [H9660] , can you assign it to me?

**Comment by HimangsKalita** [comment]
Added c program for Program to find square root without the sqrt function #10

**Comment by naveenbisht31** [comment]
Hey, I would like to contribute in **C++**. Can you assign it to me?

**Comment by nitesh-147** [comment]
Hi can you please assign it to me?

**Comment by Mrsingh782** [comment]
Hey, Ashish this side. Is someone currently working on this issue, if not can you please assign it to me.

**Comment by Saurabh-9** [comment]
hey Saurabh this side, if this issue hasn't been assign to someone so kindly assign it me

**Comment by muskaanrajput14** [comment]
Please assign it to me
